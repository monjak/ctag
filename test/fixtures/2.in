1cc

2x Goblin Guide

4cc

4 Jace, the Mind Sculptor

2cc
Tarmogoyf
