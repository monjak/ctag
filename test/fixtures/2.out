1cc

2x [c]Goblin Guide[/c]

4cc

4x [c]Jace, the Mind Sculptor[/c]

2cc
[c]Tarmogoyf[/c]
